plugins/techdocs-backend/src/techdocs/stages/publish/local.ts

```diff
@@ -18,12 +18,15 @@ import { Logger } from 'winston';
 import { Entity } from '@backstage/catalog-model';
 import { PublisherBase } from './types';
 import { resolvePackagePath } from '@backstage/backend-common';
+import { DiscoveryApi } from '@backstage/core-api';
```

Comment:
yeah this should instead be the `SingleHostDiscovery`, it's a bit confusing but what you are importing here is a frontend corresponding thing, not the backend one, they happen to have the same signature

Reply:
Done! And thanks for letting me know about the two. :)

The `discovery` that `createPlugin` passes to techdocs-backend is of the type `PluginEndpointDiscovery` which is just an implementation of `SingleHostDiscovery`.